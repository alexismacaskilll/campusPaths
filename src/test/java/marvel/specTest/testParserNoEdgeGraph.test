# Test the parser on a file that contains 3 heros from different comics

# Load a graph from tsv file that contains 3 heros from different comics
LoadGraph noEdgeGraph noEdges.tsv

# Print the heros in the graph
ListNodes noEdgeGraph

# Print the outgoing edges from each hero in the graph
ListChildren noEdgeGraph hero1
ListChildren noEdgeGraph hero2
ListChildren noEdgeGraph hero3