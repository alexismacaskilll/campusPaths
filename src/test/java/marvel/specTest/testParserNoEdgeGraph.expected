# Test the parser on a file that contains 3 heros from different comics

# Load a graph from tsv file that contains 3 heros from different comics
loaded graph noEdgeGraph

# Print the heros in the graph
noEdgeGraph contains: hero1 hero2 hero3

# Print the outgoing edges from each hero in the graph
the children of hero1 in noEdgeGraph are:
the children of hero2 in noEdgeGraph are:
the children of hero3 in noEdgeGraph are: